### PR TITLE
Improve release-notes further

### PR DESCRIPTION
**Pull request by pallavisontakke**
For docs-compliance for the upcoming release.

Disable-check: force-changelog-file
Disable-check: commit-count

### CODE REVIEW

**Review by svenklemm — APPROVED** [review]

File: .unreleased/fix_7055 [hunk 1]
```diff
@@ -1 +1 @@
-Fixes: #7055 Fix scankey where consttype is different from var
+Fixes: #7055: Fix the `scankey` for `segment by` columns, where constant type is different from variable.
```
Comment (billy-the-fish) [inline .unreleased/fix_7055:1]:
```suggestion
Fixes: #7055: Fix the `scankey` for `segment by` columns, where the type `constant` is different to `variable`.
```

File: .unreleased/fix_7064 [hunk 2]
```diff
@@ -1 +1 @@
-Fixes: #7064 Fix a bug in default orderby calc. in compression 
+Fixes: #7064: Fix the bug in default `order by` calculation in compression.
```
Comment (billy-the-fish) [inline .unreleased/fix_7064:1]:
```suggestion
Fixes: #7064: Fix the bug in the default `order by` calculation in compression.
```

File: .unreleased/fix_7074 [hunk 3]
```diff
@@ -1 +1 @@
-Fixes: #7074 Fix a bug in default segmentby calc. in compression
+Fixes: #7074: Fix the bug in default `segment by` calculation in compression.
```
Comment (billy-the-fish) [inline .unreleased/fix_7074:1]:
```suggestion
Fixes: #7074: Fix the bug in the default `segment by` calculation in compression.
```

File: .unreleased/pr_6880 [hunk 4]
```diff
@@ -1 +1 @@
-Implements: #6880 Add support for array operators to compressed DML batch filtering
+Implements: #6880: Add support for the array operators to compressed DML batch filtering.
```
Comment (billy-the-fish) [inline .unreleased/pr_6880:1]:
```suggestion
Implements: #6880: Add support for the array operators used to compressed DML batch filtering.
```

File: .unreleased/pr_6897 [hunk 5]
```diff
@@ -1 +1 @@
-Implements: #6897 Add support for replica identity on compressed hypertables
+Implements: #6897: Add support for replica identity on the compressed hypertables.
```
Comment (billy-the-fish) [inline .unreleased/pr_6897:1]:
```suggestion
Implements: #6897: Add support for replica identity on compressed hypertables.
```

File: .unreleased/pr_6918 [hunk 6]
```diff
@@ -1 +1 @@
-Implements: #6918 Remove support for PG13
+Implements: #6918: Remove the support for PG13.
```
Comment (billy-the-fish) [inline .unreleased/pr_6918:1]:
```suggestion
Implements: #6918: Remove support for PG13.
```

File: .unreleased/pr_6989 [hunk 7]
```diff
@@ -1 +1 @@
-Implements: #6989 Add support for foreign keys from plain tables to hypertables
+Implements: #6989: Add support for the foreign keys from plain tables to hypertables.
```
Comment (billy-the-fish) [inline .unreleased/pr_6989:1]:
```suggestion
Implements: #6989: Add support for foreign keys when converting plain tables to hypertables.
```

File: .unreleased/pr_7018 [hunk 8]
```diff
@@ -1 +1 @@
-Fixes: #7018 Fix search_path quoting in compression defaults function
+Fixes: #7018: Fix the `search_path` quoting in the compression defaults function.
```
Comment (billy-the-fish) [inline .unreleased/pr_7018:1]:
```suggestion
Fixes: #7018: Fix `search_path` quoting in the compression defaults function.
```

File: .unreleased/pr_7046 [hunk 9]
```diff
@@ -1 +1 @@
-Fixes: #7046 Prevent locking compressed tuples
+Fixes: #7046: Prevent locking the compressed tuples.
```
Comment (billy-the-fish) [inline .unreleased/pr_7046:1]:
```suggestion
Fixes: #7046: Prevent locking for compressed tuples.
```

File: .unreleased/pr_7048 [hunk 10]
```diff
@@ -1 +1 @@
-Implements: #7048 Add index scan to INSERT DML decompression
+Implements: #7048: Add the index scan to INSERT DML decompression.
```
Comment (billy-the-fish) [inline .unreleased/pr_7048:1]:
```suggestion
Implements: #7048: Add an index scan for INSERT DML decompression.
```

**Review by billy-the-fish — CHANGES_REQUESTED** [review]

File: .unreleased/pr_6880 (hunk 4 above)
Comment (pallavisontakke) [reply .unreleased/pr_6880]: Edited so.

File: .unreleased/pr_6897 (hunk 5 above)
Comment (pallavisontakke) [reply .unreleased/pr_6897]: Edited so.

File: .unreleased/pr_6918 (hunk 6 above)
Comment (pallavisontakke) [reply .unreleased/pr_6918]: Edited so.

File: .unreleased/pr_6880 (hunk 4 above)
Comment (pallavisontakke) [reply .unreleased/pr_6880]: Edited so.

File: .unreleased/pr_6989 (hunk 7 above)
Comment (pallavisontakke) [reply .unreleased/pr_6989]: Edited so.

File: .unreleased/pr_7018 (hunk 8 above)
Comment (pallavisontakke) [reply .unreleased/pr_7018]: Edited so.

File: .unreleased/pr_7046 (hunk 9 above)
Comment (pallavisontakke) [reply .unreleased/pr_7046]: Edited so.

File: .unreleased/pr_7048 (hunk 10 above)
Comment (pallavisontakke) [reply .unreleased/pr_7048]: Edited so.

**Review by billy-the-fish — APPROVED** [review]
LGTM

**Pull request merged**